listings_app/urls.py

```diff
@@ -16,6 +16,9 @@
 from django.contrib import admin
 from django.urls import path
 
+from . import views
+
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', views.homepage, name="homepage")
```

Comment:
This is a minor thing, but it is best to be consistent with formatting. If a file uses single quotes, you should try to use single quotes elsewhere in the file. You have mixed single and double quotes. See PEP8's recommendation here: https://www.python.org/dev/peps/pep-0008/#string-quotes

Reply:
Thanks @amyboyd, I thought about it, actually, I commented in the PR about stick to one of them but I didn't know which one, but you right, didn't realise that file was already on single quotes. I will fix :)